spec/models/edition_spec.rb

```diff
@@ -166,4 +166,32 @@
       expect(edition.change_note_html).to eq "<p><a href=\"http://example.org\">http://example.org</a></p>\n"
     end
   end
+
+  describe "#search" do
+    it "has triggers setup" do
+      expect(HairTrigger::migrations_current?).to be true
+    end
+
+    it "searches title" do
+      titles = ["Standups", "Unit Testing"]
+      titles.each_with_index do |title, index|
+        edition = Generators.valid_edition(state: "review_requested", title: title)
+        Guide.create!(latest_edition: edition, slug: "/service-manual/#{index}")
+      end
+
+      results = Edition.search("testing").map {|e| e.title}
+      expect(results).to eq ["Unit Testing"]
+    end
+
+    it "prioritises title over body" do
```

Comment:
This test shows the field weighting, but the end user can't see it because the controller loses the order when loading guides. I do not know how to solve it though.